### PR TITLE
[ISSUE #57]Uploaded Blockchain.json

### LINKED ISSUE #57: data: Blockchain roadmap
### What would you like to share?

- Create a JSON file of [Blockchain Roadmap](https://github.com/WeMakeDevs/roadmaps/tree/main/Blockchain#readme) like [this format](https://github.com/WeMakeDevs/roadmap-runner/blob/main/server/seed/data/android.json)
- Save the JSON file in [this folder server/seed/data/](https://github.com/siddhigate/roadmap-runner/tree/main/server/seed/data)

### Additional information

Note: The added roadmap will not be reflected in the UI

### PR DESCRIPTION
**Pull request by harshithasudhakar**
## Fixes Issue

This PR fixes the following issues:

closes #57 

## Changes proposed

Here comes all the changes proposed through this PR:

Adding Blockchain Roadmap

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: server/seed/data/Blockchain.json [hunk 1]
```diff
@@ -0,0 +1,604 @@
+{
+  "name": "Blockchain",
+  "sections": [
```
Comment (siddhigate) [inline server/seed/data/Blockchain.json:3]: Can you add:
```
tagline: "Comprehensive Guide to Blockchain Development",
bannerImage: "https://res.cloudinary.com/dqkl3iifo/image/upload/v1682938399/roadmaps/blockchain-roadmap_j7vyrz.jpg",
description: "Discover the world of Web3 with our comprehensive guide to blockchain development. Learn fundamentals like distributed ledgers, crypto wallets, and smart contracts. Choose your blockchain, master Solidity programming, and connect your Dapp to the frontend. Explore layer 2 scaling solutions, and the vast ecosystem of tools and SDKs available to Web3 developers. With project ideas and video tutorials from leading experts, our guide equips you to build decentralized applications today."
```
here?

Comment (harshithasudhakar) [reply server/seed/data/Blockchain.json]: Yes, I have added those now.

**Review by siddhigate — APPROVED** [review]

**Comment by siddhigate** [comment]
@harshithasudhakar Thanks for your contribution!🙌

**Comment by harshithasudhakar** [comment]
> @harshithasudhakar Thanks for your contribution!🙌

It was my pleasure to contribute!

**Pull request merged**